examples/simple-game-server/main.go

```diff
@@ -807,8 +820,12 @@ func getListValues(s *sdk.SDK, listName string) (string, error) {
 	values, err := s.Alpha().GetListValues(listName)
 	if err != nil {
 		log.Printf("Error getting List %s values: %s", listName, err)
+		return strings.Join(values, ","), err
 	}
-	return strings.Join(values, ",") + "\n", err
+	if len(values) > 0 {
+        return "VALUES: " + strings.Join(values, ",") + "\n", nil
+    }
+	return "Empty List" + strings.Join(values, ",") + "\n", err
```

Comment:
```suggestion
	return "VALUES: <none>\n", err
```
Or something like that? To be consistent with the other responses of having a "DESCRIPTOR:RESULT" format.